### PR TITLE
Added plural weekdays

### PR DESCRIPTION
**Pull request by finnjbm**
when you entered something like schedule.every(2).monday.do(func), monday would return something saying you need to use mondays, but the mondays function doesn't exist. I added the mondays-sundays functions.

### CODE REVIEW

**Comment by anthonymendez** [comment]
Really hope this is what all it would take to get it to properly schedule on certain days of the week. I was working on implementing this on my project but I realized this is something the schedule module lacks. Might look elsewhere unless this is added into schedule. But considering the last release was January 2019, and the last commit in August 2019, I have my doubts.

**Comment by MichaelSchmidt82** [comment]
I wish the author would merge this.  I really need this and I can't find a good alternative rn

**Comment by confiscar** [comment]
@finnjbm I don't think these changes actually let you schedule plural weekdays. Think there are more changes that need to be made. This PR should probably be changed or closed.

File: schedule/__init__.py [hunk 1]
```diff
@@ -322,47 +322,82 @@ def monday(self):
...
+    def saturdays(self):
+        self.start_day = 'saturday'
+        return self
+
     @property
     def sunday(self):
         if self.interval != 1:
             raise IntervalError('Use sundays instead of sunday')
         self.start_day = 'sunday'
-        return self.weeks
```
Comment (confiscar) [inline schedule/__init__.py:365]: Wait, not only does this not do what you think it does, it also breaks existing functionality. This change would totally break the` .sunday` property...

**Comment by SijmenHuizenga** [comment]
Thanks for raising your concerns! You are correct, it is not possible to schedule a task to run every n weekday. Let's look at an example:

A job is scheduled to run at every other monday (using `schedule.every(2).monday`). Let's say the program is launched on tuesday in week 1. The job will run on monday in week 3. Now for whatever reason the program is restarted on friday in week 2. Because there is no job persistence, the job will be scheduled to run on monday in week 4. 

This is weird right? A line of code that looks like it will schedule a job to run every 2 weeks will run the job will run the job only on the 4th week.

When scheduling a job to run every monday a restart at friday isn't a problem because the job will always be scheduled to run the next monday.

Schedule shouldn't be used to run jobs once in a long while. That is not what it was designed for.

I do agree that these errors could be improved.

**Pull request closed without merging**